cypress/e2e/udemy_automation-test-store/contact-us.js

```diff
@@ -1,5 +1,7 @@
 /// <reference types='cypress' />
 
+// not important change, just to force github to show this file in diff.
+
 describe('Test Contact Us for via Automation Test Store', () => {
     it('Submit a successful submission via contact us form and go back to front page', () => {
         cy.visit('https://automationteststore.com/')
```

Comment:
this is repeated here and in second test. It would be better to put this in `beforeEach` block, and also, use a baseUrl for `https://automationteststore.com` and just use `/` as go to address.